### PR TITLE
NF: Applying Decorators in Module (Workflow)

### PR DESCRIPTION
**Pull request by deka27**
## This PR is to apply keyword decorator to functions in workflow module.

This PR is dependent on the main PR, which should be merged first.

## Main PR:
- #3239

### CODE REVIEW

**Comment by skoudoro** [comment]
Can you rebase and fix the format since https://github.com/dipy/dipy/pull/3239 is merged ? Thanks!

File: dipy/workflows/mask.py [hunk 1]
```diff
@@ -13,6 +14,7 @@ class MaskFlow(Workflow):
     def get_short_name(cls):
         return "mask"
 
+    @warning_for_keywords()
     def run(self, input_files, lb, ub=np.inf, out_dir="", out_mask="mask.nii.gz"):
```
Comment (skoudoro) [inline dipy/workflows/mask.py:18]: missing `*`. any reason ?

**Review by skoudoro — CHANGES_REQUESTED** [review]
Hi @deka27, 

Can you check why it is failing? see the comment below also

**Review by itellaetxe — COMMENTED** [review]
I checked all the changes. All the function definitions that you changed seem to be correctly updated in the argument definition section. OK

On the other hand, we should check why the CIs are failing

Good work

**Review by skoudoro — APPROVED** [review]
thank you for this @deka27, merging

**Pull request merged**